release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1070,6 +1082,17 @@ With this update, the Ingress Operator no longer adds the `clientca-configmap` f
 
 * Previously, a race condition occurred between the handling of pods that had been scheduled and the pods that had been completed on a node when OVN-Kubernetes started. This condition often occurred when nodes rebooted. Consequently, the same IP was erroneously assigned to multiple pods. This update fixes the race condition so that the same IP is not assigned to multiple pods in those circumstances. (link:https://issues.redhat.com/browse/OCPBUGS-16634[*OCPBUGS-16634*])
 
+* Previously, there was an error that caused a route to be rejected due to a duplicate host claim. When this occurred, the system would mistakenly select the first route it encountered, which was not always the conflicting route. With this update, all routes for the conflicting host are first retrieved and then sorted based on their submission time. This allows the system to accurately determine and select the newest conflicting route. (link:https://issues.redhat.com/browse/OCPBUGS-16707[*OCPBUGS-16707*])
+
+* Previously, when a new ipspec-host pod was started, it would clear or remove the existing XFRM state. Consequently, it would kill existing north-south traffic policies. This issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-19817[*OCPBUGS-19817*])	
+
+* Previously, the `ovn-k8s-cni-overlay, topology:layer2` NetworkAttachmentDefinition did not work in a hosted pod when using the Kubevirt provider. Consequently, the pod did not start. This issue has been resolved, and pods can now start with an `ovn-k8s-cni-overlay` NetworkAttachmentDefinition. (link:https://issues.redhat.com/browse/OCPBUGS-22869[*OCPBUGS-22869*])	
+
+* Previously, the Azure upstream domain name server (DNS) did not comply with non-EDNS DNS queries because it returned a payload larger than 512 bytes. Because CoreDNS 1.10.1 no longer uses EDNS for upstream queries and only uses EDNS when the original client query uses EDNS, the combination would result in an overflow `servfail` error when the upstream returned a payload larger than 512 bytes for non-EDNS queries using CoreDNS 1.10.1. Consequently, upgrading from {product-title} 4.12 to 4.13 led to some DNS queries failing that previous worked.
```

Comment:
As DNS is expanded, I suggest expanding EDNS on first usage as well. EDNS is less well-known than DNS.

Reply:
Also it should be "that previously worked" - the "ly" is missing